<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeTreePanel</class>
 <widget class="QDockWidget" name="NodeTreePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Tree</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="m_newNodeButton">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../Resource/App.qrc">
              <normaloff>:/icon/Icon/project/add.png</normaloff>:/icon/Icon/project/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="m_nodeTreeWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Property</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeView" name="m_propertyTreeView">
           <property name="editTriggers">
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Signal</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="m_actionDeleteNode">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/Icon/minus.png</normaloff>:/icon/Icon/minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete Node(s)</string>
   </property>
   <property name="iconText">
    <string>Delete Node(s)</string>
   </property>
  </action>
  <action name="m_actionRenameNode">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="m_actionImportGltfScene">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/node/gltfscene.png</normaloff>:/icon/node/gltfscene.png</iconset>
   </property>
   <property name="text">
    <string>Import glTF Scene</string>
   </property>
  </action>
  <action name="m_actionAddNode">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/node/node.png</normaloff>:/icon/node/node.png</iconset>
   </property>
   <property name="text">
    <string>Add Node</string>
   </property>
  </action>
  <action name="m_actionAddChildScene">
   <property name="text">
    <string>Instance Child Scene</string>
   </property>
   <property name="toolTip">
    <string>Instance Child Scene</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../Resource/App.qrc"/>
 </resources>
 <connections/>
</ui>
